roles/cinder-control/tasks/main.yml

```diff
@@ -7,10 +7,12 @@
     name: "{{ item.name }}"
     user: "{{ item.user }}"
     cmd: "{{ item.cmd }}"
+    envs: "{{ cinder.service.envs }}"
```

Comment:
This will not work for upgrade or if encryption is enables later on. The module will skip doing anything if service is already present. We need to find smarter way to reflect the change to service.